openstates/md/bills.py

```diff
@@ -189,44 +188,43 @@ def parse_vote_pdf(self, vote_url, bill):
         text = convert_pdf(filename, type='text').decode()
         lines = text.splitlines()
 
-        if 'senate' in vote_url:
+        if 'Senate' in vote_url:
             chamber = 'upper'
         else:
             chamber = 'lower'
 
-        date_string = lines[0].split('Date:')[1].strip()
+        date_string = lines[0].split('Calendar Date:')[1].strip()
         date = datetime.datetime.strptime(date_string, "%b %d, %Y %I:%M (%p)")
 
-        pageIndex = -1
+        page_index = None
         for index, line in enumerate(lines):
             if 'Yeas' in line and 'Nays' in line:
-                pageIndex = index
+                page_index = index
                 break
 
-        yes_count = -1
-        no_count = -1
-        other_count = 0
+        vote_counts = 5*[None]
+        voteTypes = ['yes', 'no', 'not voting', 'excused', 'absent']
```

Comment:
Similar comment on the camelCase vs snake_case variable naming.

Reply:
Also, this `vote_counts` construction is a little non-standard, but it's valid syntax. And I see what you're going for.